AppImage building fails for v4.0.12

see workflow run: https://github.com/silagokth/vesyla/actions/runs/14616992582

```bash
error: this file contains an unclosed delimiter
  --> src/main.rs:72:3
   |
26 | fn main() {
   |           - unclosed delimiter
...
35 |         Err(e) => {
   |                   - this delimiter might not be properly closed...
...
38 |     };
   |     - ...as it matches this but it has different indentation
...
72 | }
   |  ^

error: could not compile `vs-manas` (bin "vs-manas") due to 1 previous error
make[2]: *** [module/vs-manas/CMakeFiles/gen-vs-manas.dir/build.make:77: cargo/vs-manas/release/vs-manas] Error 101
make[1]: *** [CMakeFiles/Makefile2:170: module/vs-manas/CMakeFiles/gen-vs-manas.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,5 +1,6 @@
 build
 env/
+.venv/
 work/
 vesyla-suite
 debug/

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,6 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+## [4.0.13] - 2025-04-23
+
+### Fixed
+
+- [#42](https://github.com/silagokth/vesyla/issues/42) Appimage does not build because of missing bracket in vs-manas
+
 ## [4.0.12] - 2025-04-23
 
 ### Added

--- a/module/vs-manas/src/gen.rs
+++ b/module/vs-manas/src/gen.rs
@@ -1,7 +1,5 @@
 use crate::instr::Instruction;
 use std::collections::HashMap;
-use std::fs::File;
-use std::io::Read;
 
 pub struct Generator {}
 

--- a/module/vs-manas/src/instr.rs
+++ b/module/vs-manas/src/instr.rs
@@ -4,7 +4,7 @@ use std::collections::HashMap;
 #[derive(Debug)]
 pub struct Instruction {
     pub name: String,
-    pub kind: String,
+    pub _kind: String,
     pub component_type: String,
     pub format: serde_json::Value,
     pub template: serde_json::Value,
@@ -43,7 +43,7 @@ impl Instruction {
 
         Instruction {
             name: name.clone(),
-            kind: kind.clone(),
+            _kind: kind.clone(),
             component_type: component_type,
             format: format,
             template: template,
@@ -51,7 +51,7 @@ impl Instruction {
         }
     }
 
-    pub fn new_from_bin(
+    pub fn _new_from_bin(
         name: &String,
         kind: &String,
         isa: &serde_json::Value,
@@ -154,7 +154,7 @@ impl Instruction {
 
         Instruction {
             name: name.clone(),
-            kind: kind.clone(),
+            _kind: kind.clone(),
             component_type: component_type,
             format: format,
             template: template,

--- a/module/vs-manas/src/main.rs
+++ b/module/vs-manas/src/main.rs
@@ -5,7 +5,7 @@ mod parser;
 use clap::Parser as clappar;
 use core::panic;
 use gen::Generator;
-use log::{debug, error, info, trace, warn};
+use log::{error, info};
 use parser::Parser;
 use std::fs;
 use std::path::Path;
@@ -35,6 +35,7 @@ fn main() {
         Err(e) => {
             error!("{}", e);
             std::process::exit(1);
+        }
     };
 
     info!("Assembler started: {:?}", args.asm);

--- a/module/vs-manas/src/parser.rs
+++ b/module/vs-manas/src/parser.rs
@@ -128,8 +128,8 @@ impl Parser {
         let mut pc_table: HashMap<String, HashMap<String, i32>> = HashMap::new();
         let mut cell_instr_counter: HashMap<String, i32> = HashMap::new();
         let prog = prog.unwrap();
-        let mut x = 0;
-        let mut y = 0;
+        let mut x;
+        let mut y;
         let mut label = String::new();
         for record in prog.clone().records {
             if record.name == "cell" {

